docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx

```diff
@@ -169,7 +169,7 @@ Customers can also configure the recordings to be stored in their cloud storage.
 
 [Create Room](/server-side/v2/Rooms/create-via-api)
 
-Recording mp4 will be placed at - `s3://<location>/<prefix>/beam/<room_id>/<start_date>/Rec-<room_id>-<epoch>.mp4`. Example `s3://test-bucket/test-prefix/beam/61101349d0f56e684b608c08/20210721/Rec-61101349d0f56e684b608c08-1626898422000.mp4`
+Recording mp4 will be placed at - `<upload_type>://<location>/<prefix>/beam/<room_id>/<start_date>/Rec-<room_id>-<epoch>.mp4`. Example `s3://test-bucket/test-prefix/beam/61101349d0f56e684b608c08/20210721/Rec-61101349d0f56e684b608c08-1626898422000.mp4`
```

Comment:
Should be "type" instead of "upload_type"?

Reply:
This document doesn't mention about an upload_info object that has `type`, `location` etc. So thought it'll be straight forward to understand if we write it as `upload_type`.

Reply:
Let go with this for now